LeetcodeProblems/Algorithms/2428.Hour_Glass_Sum.js

```diff
@@ -0,0 +1,33 @@
+/*
+2428. Maximum Sum of an Hourglass
+
+You are given an m x n integer matrix grid.
+We define an hourglass as a part of the matrix with the following form:
```

Comment:
Could you add an example of the matrix here? It could be with text if the original problem use images.